src/zimscraperlib/zim/creator.py

```diff
@@ -177,10 +177,15 @@ def validate_metadata(
     def add_metadata(
         self,
         name: str,
-        content: Union[str, bytes, datetime.date, datetime.datetime],
+        content: Union[str, bytes, datetime.date, datetime.datetime, Iterable[str]],
         mimetype: str = "text/plain;charset=UTF-8",
     ):
         self.validate_metadata(name, content)
+        # handle necessary type conversions
```

Comment:
Please remove all comments ; code is almost self explanatory and your PR will document the rest. We prefer to avoid to clutter the code with comments unless the behavior is really weird or something unexpected has to be mentioned.